Sources/TuistKit/Services/TestService.swift

```diff
@@ -170,20 +170,20 @@ final class TestService {
             ) {
                 try FileHandler.shared.createFolder(cacheDirectoriesProvider.cacheDirectory(for: .tests))
             }
-
-            // Saving hashes from `testsCacheTemporaryDirectory` to `testsCacheDirectory` after all the tests have run successfully
-            try FileHandler.shared
-                .contentsOfDirectory(testsCacheTemporaryDirectory.path)
-                .forEach { hashPath in
-                    let destination = cacheDirectoriesProvider.cacheDirectory(for: .tests).appending(component: hashPath.basename)
-                    guard !FileHandler.shared.exists(destination) else { return }
-                    try FileHandler.shared.move(
-                        from: hashPath,
-                        to: destination
-                    )
-                }
         }
 
+        // Saving hashes from `testsCacheTemporaryDirectory` to `testsCacheDirectory` after all the tests have run successfully
+        try FileHandler.shared
```

Comment:
Could you also make a test that would fail without this fix? Seems important enough to make sure there is no regression.

Reply:
Will do. Think this would be a good fit for an acceptance test? Or mock out FileManager in a unit test?

Reply:
The unit test might suffice here but I'll let you call the shot here 🙂

Reply:
Unit test written 👍